client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -406,19 +412,87 @@ describe('FeedbackGeneratorService', function() {
       ].join(''));
       expect(paragraphs[1].isErrorParagraph()).toBe(true);
       expect(paragraphs[1].getContent()).toBe(
-        'ZeroDivisionError: integer division or modulo by zero on a line ' +
-        'in the test code');
+        'ZeroDivisionError: integer division or modulo by zero');
     });
   });
 
-  describe('_getHumanReadableRuntimeFeedback', function() {
-    it([
-      'should return the correct feedback string if the code throws an ',
+  describe('_getSyntaxRuntimeFeedback', function() {
+    it(['should return the correct feedback string if ',
+      'an undeclared name is used'].join(''), function() {
+      var errorString = "NameError: global name foo is not defined";
+      var feedbackString =
+          FeedbackGeneratorService._getFriendlySyntaxFeedback(
+            errorString, LANGUAGE_PYTHON);
+      expect(feedbackString).toEqual([
+        '<code>foo</code> appears to be a name that was not previously ',
+        'declared or defined. E.g., if <code>foo</code> is a variable to ',
+        'hold a string, <code>foo = \'\'</code> would declare ',
+        '<code>foo</code> as a string variable. Similarly, ',
+        '<code>foo = []</code> declares <code>foo</code> as an array ',
+        'variable. Another possibility is that <code>foo</code> was ',
+        'misspelled or uses incorrect capitalization.'].join(''));
+    });
+    it(['should return the correct feedback string if ',
```

Comment:
Add newlines between it() blocks, similar to the rest of the file.

Reply:
ACK Fixed in upcoming commit.